### PR TITLE
chore: Rename prisma/e2e-tests to ecosystem-tests

### PR DESCRIPTION
**Pull request by janpio**
Note that this changes the release relevant ENV var, so requires documentation update right after merging and consideration on next release.

### CODE REVIEW

File: packages/client/README.md [hunk 1]
```diff
@@ -23,5 +23,5 @@ Refer to our [contribution guidelines](https://github.com/prisma/prisma/blob/mai
 
 - Prisma Tests Status:  
   [![Build status](https://badge.buildkite.com/590e1981074b70961362481ad8319a831b44a38c5d468d6408.svg?branch=master)](https://buildkite.com/prisma/prisma2-test)
-- E2E Tests Status:  
-  [![Actions Status](https://github.com/prisma/prisma2-e2e-tests/workflows/test/badge.svg)](https://github.com/prisma/prisma2-e2e-tests/actions)
+- Ecosystem Tests Status:  
+  [![Actions Status](https://github.com/prisma/prisma2-ecosystem-tests/workflows/test/badge.svg)](https://github.com/prisma/prisma2-ecosystem-tests/actions)
```
Comment (Jolg42) [inline packages/client/README.md:27]:
```suggestion
  [![Actions Status](https://github.com/prisma/ecosystem-tests/workflows/test/badge.svg)](https://github.com/prisma/prisma2-ecosystem-tests/actions)
```

File: README.md [hunk 2]
```diff
@@ -256,5 +256,5 @@ Refer to our [contribution guidelines](https://github.com/prisma/prisma/blob/mai
 
 - Prisma Tests Status:  
   [![Build status](https://badge.buildkite.com/590e1981074b70961362481ad8319a831b44a38c5d468d6408.svg?branch=main)](https://buildkite.com/prisma/prisma2-test)
-- E2E Tests Status:  
-  [![Actions Status](https://github.com/prisma/prisma2-e2e-tests/workflows/test/badge.svg)](https://github.com/prisma/prisma2-e2e-tests/actions)
+- Ecosystem Tests Status:  
+  [![Actions Status](https://github.com/prisma/prisma2-ecosystem-tests/workflows/test/badge.svg)](https://github.com/prisma/prisma2-ecosystem-tests/actions)
```
Comment (Jolg42) [inline README.md:260]:
```suggestion
  [![Actions Status](https://github.com/prisma/ecosystem-tests/workflows/test/badge.svg)](https://github.com/prisma/ecosystem-tests/actions)
```

**Review by Jolg42 — APPROVED** [review]
Looks good, see suggestions for some broken links

File: TESTING.md [hunk 3]
```diff
@@ -64,11 +64,11 @@ pnpm run test <fileNamePattern> -- -u
 
 Something is broken? You built a new feature? It's time to write a test! But where?
 
-Everything related to working with specific frameworks like Next.js or deploying to Netlify should be covered by an [End-to-End Test](https://github.com/prisma/e2e-tests).
+Everything related to working with specific frameworks like Next.js or deploying to Netlify should be covered by an [End-to-End Test](https://github.com/prisma/ecosystem-tests).
```
Comment (aqrln) [inline TESTING.md:67]:
```suggestion
Everything related to working with specific frameworks like Next.js or deploying to Netlify should be covered by an [Ecosystem Test](https://github.com/prisma/ecosystem-tests).
```

**Comment by Jolg42** [comment]
@janpio Let's merge?

**Comment by janpio** [comment]
Replaced all mentions of `SKIP_E2E_CHECK` in Notion.

**Pull request merged**